org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -528,4 +531,130 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 		#{"**/*.xtend","**/*.mwe2"}
 	}
 	
+	/**
+	 * @since 2.11
+	 */
+	def private getWfLaunchConfigFile() {
```

Comment:
They are not needed for private methods, I consider them in this case just as annotation that the code was added with 2.11. Usually they are used to specify that *API* was added with a version.
API tooling does not work for me right now. I asked already on Slack how we handle API in Xtend without answer yet.